frameworks/helloworld/src/main/dist/examples/secrets.yml

```diff
@@ -0,0 +1,83 @@
+name: hello-world
+scheduler:
+  principal: {{SERVICE_PRINCIPAL}}
+  user: {{SERVICE_USER}}
+pods:
+  hello:
+    container:
+      image-name: ubuntu:14.04
+    count: {{HELLO_COUNT}}
+    placement: {{HELLO_PLACEMENT}}
+    secrets:
+      s1:
```

Comment:
@mgummelt YAML Option 1 in design doc is probably what you are looking for. This one (Option 2) was selected (comments were: more YAML like, and conforms better to existing YAML)